spec/features/saml/loa1_sso_spec.rb

```diff
@@ -40,6 +40,25 @@
       )
       expect(page).to_not have_css('.accordion-header')
     end
+
+    it 'allows user to view recovery code via profile' do
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
+      saml_authn_request = auth_request.create(saml_settings)
+      user = create(:user, :with_phone)
+      code = RecoveryCodeGenerator.new(user).create
+
+      visit saml_authn_request
+      sign_in_and_require_viewing_recovery_code(user)
+      click_on(t('shared.nav_auth.my_account'))
+      click_on(t('profile.links.regenerate_recovery_code'))
+
+      expect(current_path).to eq manage_recovery_code_path
+
+      click_acknowledge_recovery_code
+      enter_recovery_code(code: code)
```

Comment:
Sounds like you are making 2 points:

1. This spec needs js to be turned on in order to test the *real* behavior. Right now, real behavior does not redirect as expected.

2. This PR is making an undesired product change.

More than happy to update and fix 1, but let's get feedback on the product side re: point 2 first :) cc @andrewhughey